nss-day-cohort-30/bangazon-site-plucky-pikes
# Expiration date

# Description
Added Expiration Date to Payment Types and custom validation to ensure that the expiration date entered by the user is in the future. The addition of the Expiration Date field requires a database migration.

## Type of change
Please delete options that are not relevant.

# Testing Instructions for Change Made
1. `git fetch --all`
1. `git checkout expiration-date`
1. Panya and I will talk through doing the database migration
1. once the database migration is finished, start the debugger and go to the Bangazon site
1. go to the account manager page by clicking the user name in the nav bar
1. click manage payment types and add a new payment type
1. an expiration date field should now appear at the bottom of the form
1. enter a date in the past and try to submit the form; the form should not submit
1. enter a date in the future and submit the form; the form should submit, return the user to the Payment Type index page, and the expiration date should appear on the index view.
1. self high-five


# Checklist:
